src/tables/Connectors.tsx

```diff
@@ -0,0 +1,117 @@
+import {
+    Box,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableHead,
+    TableRow,
+    Typography,
+} from '@mui/material';
+import useConnectors from 'hooks/useConnectors';
+import { FormattedDate, FormattedMessage, useIntl } from 'react-intl';
+
+function ConnectorsTable() {
+    const { connectors, isFetchingConnectors, fetchingConnectorsError } =
+        useConnectors();
+
+    const intl = useIntl();
+
+    const columns = [
+        {
+            field: 'attributes.name',
+            headerIntlKey: 'data.name',
+        },
+        {
+            field: 'attributes.description',
+            headerIntlKey: 'data.description',
+        },
+        {
+            field: 'attributes.type',
+            headerIntlKey: 'data.type',
+        },
+        {
+            field: 'attributes.maintainer',
+            headerIntlKey: 'data.maintainer',
+        },
+        {
+            field: 'attributes.updated_at',
+            headerIntlKey: 'data.updated_at',
+        },
+    ];
+
+    return (
+        <Box
+            sx={{
+                mx: 2,
+            }}
+        >
+            {isFetchingConnectors ? (
+                <FormattedMessage id="common.loading" />
+            ) : null}
+
+            {fetchingConnectorsError ? { fetchingConnectorsError } : null}
+
+            {connectors.length > 0 ? (
+                <TableContainer component={Box}>
+                    <Typography>
+                        <FormattedMessage id="terms.connectors" />
+                    </Typography>
+                    <Table
+                        sx={{ minWidth: 350 }}
+                        aria-label={intl.formatMessage({
+                            id: 'connectors.title',
+                        })}
+                    >
+                        <TableHead>
+                            <TableRow
+                                sx={{
+                                    background: (theme) =>
+                                        theme.palette.background.default,
+                                }}
+                            >
+                                {columns.map((column, index) => {
+                                    return (
+                                        <TableCell
+                                            key={`${column.field}-${index}`}
+                                        >
+                                            <FormattedMessage
+                                                id={column.headerIntlKey}
+                                            />
+                                        </TableCell>
+                                    );
+                                })}
+                            </TableRow>
+                        </TableHead>
```

Comment:
Styled the header with the default background color... not sure if this is the best pattern but it made it work. We should check back in on this later.